.gitignore

```diff
@@ -1,5 +1,4 @@
 .vagrant
-Berksfile.lock
```

Comment:
This is a respectable choice (to ignore Berksfile.lock and Gemfile.lock). Just wanted to mention it in the off chance it was an unintended change.